astroplan/plots/time_dependent.py

```diff
@@ -145,40 +147,49 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
                       'to use a scalar. (Or maybe a list with length > 1?).',
                       PlotWarning)
 
-    # Calculate airmass
-    airmass = observer.altaz(time, target).secz
-    # Mask out nonsense airmasses
-    masked_airmass = np.ma.array(airmass, mask=airmass < 1)
+    if not isinstance(targets, list):
```

Comment:
What about testing for `collections.Sequence` or `collections.Iterable` here? For example the current implementation would wrap a tuple of targets as list. So I would end up with a tuple of a list of targets.

I don't think that will happen often but it's probably more user-friendly.